symbolic-symcache/src/lib.rs

```diff
@@ -1,4 +1,101 @@
-//! Provides SymCache support.
+//! Provides SymCache support. This includes a reader and writer for the binary format, as well as
+//! helper traits and functions to apply transformations to debugging symbols before they are
+//! written to the SymCache.
+//!
+//! # Structure of a SymCache
+//!
+//! A SymCache (version 7) contains the following primary kinds of data, written in the following
+//! order:
+//!
+//! 1. Files
+//! 2. Functions
+//! 3. Source Locations
+//! 4. Address Ranges
+//! 5. String Data
```

Comment:
All these parts are detailed below, only this one isn't.  I don't consider this blocking though, as this was already the case and this is just moving things around.  But if you feel like doing a followup PR to fix that... 😉

Reply:
i'll see what i can do about adding a section for string data in a followup, especially since i added this entry in to begin with 👍